### DISCUSSION TITLE
Add index context in update body

### DISCUSSION THREAD

**Discussion by bidoubiwa** [meilisearch/product#105] [Feedback & Feature Proposal]
**Is your feature request related to a problem? Please describe.**
No way of knowing in which index documents are being added.

**Describe the solution you'd like**
In the return body of an update there is a sub object that describes the action: 
```json
{
  "status": "processed",
  "updateId": 1,
  "type": {
    "name": "DocumentsAddition",
    "number": 4
  },
  "duration": 0.076980613,
  "enqueuedAt": "2019-12-07T21:16:09.623944Z",
  "processedAt": "2019-12-07T21:16:09.703509Z"
}
```
I would like the index uid related to this `DocumentsAddition` in order to know on multiple index which has all its documents properly indexed

**Comment by curquiza** [comment]
@gmourier this can interest you
